fail2ban/server/action.py

```diff
@@ -532,7 +532,7 @@ def _processCmd(self, cmd, aInfo = None):
 		return self.executeCmd(realCmd, self.timeout)
 
 	@staticmethod
-	def executeCmd(realCmd, timeout=60):
+	def executeCmd(realCmd, timeout=60, testCmd=False):
```

Comment:
testCmd is not addressed to test environment but when a command is used as a test condition to something; in this case, with ignorecommand, which returns a nonzero or zero depending on which IP should be ignored or not.

Using this var, you are telling executeCmd to do not log the nonzero return as an ERROR, because it is expected. This closes #1194 .